cocos2d/core/platform/CCInputManager.js

```diff
@@ -217,13 +217,19 @@ let inputManager = {
                 ccTouch._setPrevPoint(selTouch._prevPoint);
                 ccTouch._lastModified = now;
                 handleTouches.push(ccTouch);
+
+                // event will be distributed multiple times on opharmony platform, requires filtering by windowId
+                if(globalThis.oh && selTouch.windowId == selTouch._id) {
```

Comment:
Why can't we do this in eventmanager?

Reply:
Is it a real window id or just a tag to distinguish touches? If it is a tag, then please use a more meaningful name.

Reply:
> Why can't we do this in eventmanager?

Because windowId cannot be obtained in eventmanager.

Reply:
> Is it a real window id or just a tag to distinguish touches? If it is a tag, then please use a more meaningful name.

 it is a tag. I have modified the name already.